src/main/resources/assets/arcanuscontinuum/lang/fr_fr.json

```
@@ -0,0 +1,334 @@
+
+	"advancements.arcanuscontinuum.root.title": "Arcanus: Continuum",
+	"advancements.arcanuscontinuum.root.description": "[argument spirituel pour vous faire jouer au jeu que auquel vous jouez déjà]",
+	"advancements.arcanuscontinuum.a_magical_crystal.title": "Un Crystal Magique",
+	"advancements.arcanuscontinuum.a_magical_crystal.description": "Obtenez un Amas d'Améthyste",
+	"advancements.arcanuscontinuum.arcane_artifice.title": "Arcane Artifice",
+	"advancements.arcanuscontinuum.arcane_artifice.description": "Fabriquez un Etablie Arcanique",
+	"advancements.arcanuscontinuum.meet_and_greet.title": "Rencontre Magique",
+	"advancements.arcanuscontinuum.meet_and_greet.description": "Trouvez une Tour de Sorcier et rencontrer son occupant",
+	"advancements.arcanuscontinuum.wizard_level_1.title": "Premier Pas",
+	"advancements.arcanuscontinuum.wizard_level_1.description": "Obtenez votre premier Parchemin de Connaissance d'un Sorcier",
+	"advancements.arcanuscontinuum.wizard_level_2.title": "Sorcier de Niveau 2",
+	"advancements.arcanuscontinuum.wizard_level_2.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 2",
+	"advancements.arcanuscontinuum.wizard_level_3.title": "Sorcier de Niveau 3",
+	"advancements.arcanuscontinuum.wizard_level_3.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 3",
+	"advancements.arcanuscontinuum.wizard_level_4.title": "Sorcier de Niveau 4",
+	"advancements.arcanuscontinuum.wizard_level_4.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 4",
+	"advancements.arcanuscontinuum.wizard_level_5.title": "Sorcier de Niveau 5",
+	"advancements.arcanuscontinuum.wizard_level_5.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 5",
+	"advancements.arcanuscontinuum.wizard_level_6.title": "Sorcier de Niveau 6",
+	"advancements.arcanuscontinuum.wizard_level_6.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 6",
+	"advancements.arcanuscontinuum.wizard_level_7.title": "Sorcier de Niveau 7",
+	"advancements.arcanuscontinuum.wizard_level_7.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 7",
+	"advancements.arcanuscontinuum.wizard_level_8.title": "Sorcier de Niveau 8",
+	"advancements.arcanuscontinuum.wizard_level_8.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 8",
+	"advancements.arcanuscontinuum.wizard_level_9.title": "Sorcier de Niveau 9",
+	"advancements.arcanuscontinuum.wizard_level_9.description": "Utilisez un Parchemin de Connaissance afin d'atteindre le niveau 9",
+	"advancements.arcanuscontinuum.wizard_level_10.title": "Maître des Arcanes",
+	"advancements.arcanuscontinuum.wizard_level_10.description": "Atteindre une puissance magique maximale!",
+
+	"text.arcanuscontinuum.disabled_component": "Ce sort contient un ou plusieurs composants de sort désactivées. Abandon du sort.",
```

```suggestion
	"text.arcanuscontinuum.disabled_component": "Ce sort contient un ou plusieurs composantes de sort désactivées. Abandon du sort.",
```
Since you wrote "désactivées", I assume you meant to write "composantes", but making them both gramatically masculine could work too (with slightly different implications, "composante" sounds more abstract, which IMO works well for a spell).